content/technotes/2020-02-26-tic-ghactions.Rmd

```diff
@@ -50,42 +50,86 @@ Furthermore {tic} aims to simplify the tedious process of setting up deployment
 ("Deployment" means that a CI run is allowed to perform a `git` push to the repository.)
 This is achieved by its helper packages such as {circle}, {ghactions} or {travis} which perform API calls to the respective CI providers.
```

Comment:
in our style guide, package names need to be written without curly braces or any special formatting. :-)

Reply:
Thanks, I'll consult it again before submitting for review.